### PR TITLE
NYTimes case and death data

### PR DESCRIPTION
**Pull request by cc7768**

### CODE REVIEW

File: schemas/usgeo_tables.sql [hunk 1]
```diff
@@ -3,7 +3,7 @@ DROP TABLE IF EXISTS meta.us_fips CASCADE;
 CREATE TABLE meta.us_fips (
     "id" SERIAL PRIMARY KEY,
     "fips" BIGINT,
-    "name" VARCHAR(30),
```
Comment (sglyon) [inline schemas/usgeo_tables.sql:6]: What is the purpose for this change?

**Review by sglyon — COMMENTED** [review]
Looks good other than one comment about changing to TEXT in the meta.us_fips table

File: schemas/usgeo_tables.sql (hunk 1 above)
Comment (cc7768) [reply schemas/usgeo_tables.sql:6]: After I removed the fips filter (which ignored things like territories), there was at least one name that was longer than 30 characters. It is a relatively small table so I didn't think the `TEXT` type would cause any issues.

Comment (sglyon) [reply schemas/usgeo_tables.sql:6]: makes sense -- This is ok with me!
